src/supportedPlatforms.js

```diff
@@ -9,6 +9,10 @@ const supportedPlatforms = {
     storeUrl:
       'https://addons.mozilla.org/en-US/firefox/addon/cozy-personal-cloud'
   }
+  // safari: {
+  //   label: 'Safari',
+  //   storeUrl: 'macappstore://itunes.apple.com/app/id1504487449?mt=12'
+  // }
```

Comment:
can we uncomment these lines and use a flag? Like that we can deploy the app in production and just set the flag value when the app is validated?

Reply:
Yes but I have to refactor the code since the code is executed at module level.

Reply:
Done in 7edfeca